<!DOCTYPE html>
<html lang="en-US">
<head>

    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="theme" content="hugo-academic-group">

    
    

    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <script src="https://blachlylab.github.io/js/hugo-academic-group.js"></script>

    <link rel="stylesheet" href="https://blachlylab.github.io/css/bootstrap.min.css">
    <script src="https://blachlylab.github.io/js/bootstrap.min.js"></script>  

    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.6.0/styles/default.min.css">

    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/github-fork-ribbon-css/0.2.2/gh-fork-ribbon.min.css" />
    
    
    <script src="https://blachlylab.github.io/js/highlight.pack.js"></script>
    <script>hljs.initHighlightingOnLoad();</script>


    <link rel="stylesheet" href="https://blachlylab.github.io/css/font-awesome.min.css">
    <link rel="stylesheet" href="https://blachlylab.github.io/css/academicons.min.css">
    <link rel="stylesheet" href="https://blachlylab.github.io/css/biaslab-font.css">
    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:100,300,400,700|Merriweather:100,400,700|Roboto+Mono">
    <link rel="stylesheet" href="https://blachlylab.github.io/css/hugo-academic-group.css">

    


    <link rel="shortcut icon" href="https://blachlylab.github.io/img/favicon.ico" type="image/x-icon">
    <link rel="canonical" href="https://blachlylab.github.io/publication/blood-2019-130856/">

    <title>NPM1 mutations Using Deep Amplicon Sequencing and Broad Next Generation Sequencing at the Time of Complete Remission Is Informative to Predicting Risk of Relapse Following Intensive Chemotherapy | Blachly Lab</title>

</head>

<body>
<div class="home-anchor" id="home"></div>



<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
    <div class="container">

        
        <div class="navbar-header">
            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".navbar-collapse" aria-expanded="false">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>

            <div class="navbar-brand">
                
                    <a class="logo" href="">
                        <img src="https://blachlylab.github.io/img/lab_logo.png" alt="University logo"></img>
                    </a>
                
                
                 
                    
                    
                        <a href="">Blachly Lab</a>
                    
    
                
            </div>
        </div>

        
        <div class="collapse navbar-collapse" id="#navbar-collapse-1">

            
            <ul class="nav navbar-nav navbar-right">
                
                    
                        <li class="nav-item"><a data-scroll href="https://blachlylab.github.io/#top">Home</a></li>
                    
                
                    
                        <li class="nav-item"><a data-scroll href="https://blachlylab.github.io/research">Research</a></li>
                    
                
                    
                        <li class="nav-item"><a data-scroll href="https://blachlylab.github.io/#publications">Publications</a></li>
                    
                
                    
                        <li class="nav-item"><a data-scroll href="https://blachlylab.github.io/#projects">Projects</a></li>
                    
                
                    
                        <li class="nav-item"><a data-scroll href="https://blachlylab.github.io/#members">Members</a></li>
                    
                
                    
                        <li class="nav-item"><a data-scroll href="https://blachlylab.github.io/#posts">Blog</a></li>
                    
                
                        
                        
                        <li class="dropdown">
                            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Join us <span class="caret"></span></a>
                            <ul class="dropdown-menu">
                              
                                <li><a href="">Internships</a></li>
                              
                                <li><a href="">Vacancies</a></li>
                              
                            </ul>
                          </li>
                    
                
                        
                        
                        <li class="dropdown">
                            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Software <span class="caret"></span></a>
                            <ul class="dropdown-menu">
                              
                                <li><a href="https://github.com/blachlylab/dhtslib">dhtslib</a></li>
                              
                                <li><a href="https://github.com/blachlylab/fade">fade</a></li>
                              
                                <li><a href="https://github.com/blachlylab/fishnet">fishnet</a></li>
                              
                                <li><a href="https://github.com/blachlylab/maskerade">maskerade</a></li>
                              
                                <li><a href="https://github.com/blachlylab/mucor3">mucor3</a></li>
                              
                                <li><a href="https://github.com/blachlylab/nopilesum">nopilesum</a></li>
                              
                                <li><a href="https://github.com/blachlylab/recontig">recontig</a></li>
                              
                                <li><a href="https://github.com/blachlylab/SABER">SABER</a></li>
                              
                                <li><a href="https://github.com/blachlylab/swiftover">swiftover</a></li>
                              
                            </ul>
                          </li>
                    
                
                    
                        <li class="nav-item"><a data-scroll href="https://blachlylab.github.io/#contact">Contact</a></li>
                    
                
                
            </ul>

        </div>
    </div>
</nav>



<div class="container">

    <div class="pub" itemscope itemtype="http://schema.org/CreativeWork">

        <div class="row">
            <div class="col-sm-12">
                <div class="pub-title">
                    <h1 itemprop="name">NPM1 mutations Using Deep Amplicon Sequencing and Broad Next Generation Sequencing at the Time of Complete Remission Is Informative to Predicting Risk of Relapse Following Intensive Chemotherapy</h1>
                </div>
            </div>
        </div>
        
        <div class="row">
            <div class="col-sm-12">
                <div class="pub-authors" itemprop="author">
                    
<div itemprop="author">
        
            
                
<span class="author-name">
    
        Bhavana Bhatnagar
    
</span>,
            
                
<span class="author-name">
    
        Shelley Orwick
    
</span>,
            
                
<span class="author-name">
    
        Nyla A. Heerema
    
</span>,
            
                
<span class="author-name">
    
        Alison R. Walker
    
</span>,
            
                
<span class="author-name">
    
        Alice S. Mims
    
</span>,
            
                
<span class="author-name">
    
        James S. Blachly
    
</span>,
            
                
<span class="author-name">
    
        Karilyn Larkin
    
</span>,
            
                
<span class="author-name">
    
        Nicole Grieselhuber
    
</span>,
            
                
<span class="author-name">
    
        Gregory K. Behbehani
    
</span>,
            
                
<span class="author-name">
    
        Apollinaire Ngankeu
    
</span>,
            
                
<span class="author-name">
    
        Charles Thomas Gregory
    
</span>,
            
            
                 
<span class="author-name">
    
        John C. Byrd
    
</span>
            
        
</div>

                </div>
            </div>
        </div>

        
        
        <div class="row">
            <div class="col-sm-12">
                <h3>Abstract</h3>
            </div>
        </div>
        
        <div class="row">
            <div class="col-sm-12">
                <p class="pub-abstract" itemprop="text">Introduction: NPM1 gene mutations are a common molecular aberration in acute myeloid leukemia (AML). In the absence of concurrent high FLT3-ITD ratio mutations (&gt;0.5), NPM1 mutations typically associate with higher complete remission (CR) rates following intensive induction chemotherapy. NPM1 mutations have been shown to be stable markers of persistent disease or impending relapse during CR or complete remission with incomplete count recovery (CRi). Given the clinical implications that persistent NPM1 mutations can have during CR/CRi, we used Deep Amplicon sequencing on CR/CRi bone marrow (BM) samples collected from adult de novoNPM1-mutated AML patients to determine the ability of NPM1 mutations at both a high and lower sensitivity next generation sequencing methods and also the presence of additional clonal abnormalities on relapse risk. Methods: We performed targeted next generation sequencing (NGS) analysis in addition to NPM1 Deep Amplicon sequencing on paired BM or blood samples collected from 38 newly diagnosed NPM1-mutated AML patients during CR/CRi after successful induction (1-2 courses of 7 + 3) and, if available, at relapse. NPM1 mutated NGS libraries were prepared using a KAPA HyperPlus Kit (Roche, Pleasanton, CA) and xGen Lockdown Probes (IDT, Coralville, IA). Libraries were sequenced using the Illumina HiSeq 4000 (Illumina, San Diego, CA). GATK&rsquo;s MuTect2 was used to perform variant calling. Variant allele frequency (VAF) cut-off for the NGS panel was 0.05 (5%) with the exception of hotspot variants in IDH1 (R132) and IDH2 (R140) where variants detected to a level of 0.01 (1%) were included. The VAF cut off used for NPM1 Deep Amplicon sequencing was 0.00012 (0.012%). Results: Targeted NGS analysis and NPM1 Deep Amplicon sequencing had exceptional concordance at the level of detection of VAF= 0.05 (Figure 1). Of 38 patients, 23 patients had undetectable NPM1 mutations as analyzed through NPM1 Deep Amplicon sequencing of whom 9 (38.1%) relapsed. In contrast, 15 patients were positive by NPM1 Deep Amplicon sequencing and 9 (60%) relapsed. Only 4 patients had detectable persistent NPM1 mutations after induction according to both detection techniques and two of these relapsed. We next examined the potential impact of clearing both NPM1 mutation and co-occurring mutations together on relapses (Figure 2). A total of 15 patients cleared all of their clonal abnormalities and 5 (27%) relapsed. In contrast, of the 23 patients who did not clear the NPM1 mutation and/or another co-occurring mutation at remission, 14 (61%) have relapsed. Eleven of the relapsed patients had relapse samples available of whom all had persistent NPM1 mutation at this time. Paired CR/CRi and relapsed samples showed acquisition or recurrence of several other mutations, most notably FLT3-ITD, IDH1, and IDH2 which are all targetable with small molecule therapeutics. Conclusions: The use of Deep Amplicon sequencing to identify NPM1 mutations at a lower detection threshold compared to standard NGS techniques was more sensitive, but did not appear to fully inform relapse rates in NPM1-mutated AML patients after receipt of induction therapy. The appearance of other AML-associated mutations, identified together with NPM1 at time of remission, was more frequent among patients relapsing. These pilot data provide support for concurrent assessment of Deep Amplicon sequencing together with a broad standard NGS AML mutational assay to further enhance risk stratification of NPM1-mutated patients. Additionally, while NPM1 clones are present in all patients examined at the time of relapse, persistence or development of targetable clones justifies repeat broad NGS sequencing at this time. Figure Disclosures Bhatnagar: Novartis and Astellas: Consultancy, Honoraria; Cell Therapeutics, Inc.: Other: Research support; Karyopharm Therapeutics: Other: Research support. Mims:Agios Pharmaceuticals: Membership on an entity&rsquo;s Board of Directors or advisory committees; Astellas Pharmaceuticals: Membership on an entity&rsquo;s Board of Directors or advisory committees; Jazz Pharmaceuticals: Membership on an entity&rsquo;s Board of Directors or advisory committees; Abbvie: Membership on an entity&rsquo;s Board of Directors or advisory committees; PTC Therapeutics: Membership on an entity&rsquo;s Board of Directors or advisory committees. Behbehani:Fluidigm corporation: Other: Travel funding. Byrd:Novartis: Other: Travel Expenses, Speakers Bureau; TG Therapeutics: Other: Travel Expenses, Research Funding, Speakers Bureau; Genentech: Research Funding; Pharmacyclics LLC, an AbbVie Company: Other: Travel Expenses, Research Funding, Speakers Bureau; BeiGene: Research Funding; Janssen: Consultancy, Other: Travel Expenses, Research Funding, Speakers Bureau; Acerta: Research Funding; Gilead: Other: Travel Expenses, Research Funding, Speakers Bureau; Ohio State University: Patents &amp; Royalties: OSU-2S.</p>
            </div>
        </div>

        <div class="row">
            <div class="col-sm-12">
                <div class="row">
                    <div class="col-xs-12 col-sm-3 pub-row-heading">Publication</div>
                    <div class="col-xs-12 col-sm-9">NPM1 mutations Using Deep Amplicon Sequencing and Broad Next Generation Sequencing at the Time of Complete Remission Is Informative to Predicting Risk of Relapse Following Intensive Chemotherapy</div>
                </div>
            </div>
        </div>
        
        <div class="visible-xs space-below"></div>

        

        <div class="row">
            <div class="col-sm-12">
                <div class="row">
                    <div class="col-xs-12 col-sm-3 pub-row-heading">Date</div>
                    <div class="col-xs-12 col-sm-9" itemprop="datePublished">
                        
                        November, 2019
                    </div>
                </div>
            </div>
        </div>

        <div class="visible-xs space-below"></div>

        <div class="row" style="padding-top: 10px">
            <div class="col-sm-12">
                <div class="row">
                    <div class="col-xs-12 col-sm-3 pub-row-heading" style="line-height:34px;">Links</div>
                    <div class="col-xs-12 col-sm-9">

                        




<a class="btn btn-primary btn-outline" href="https://ashpublications.org/blood/article/134/Supplement_1/1329/427367/NPM1-mutations-Using-Deep-Amplicon-Sequencing-and">PDF</a>




<a class="btn btn-primary btn-outline" href="https://blachlylab.github.io/project/research">Project</a>





                    </div>
                </div>
            </div>
        </div>
        <div class="visible-xs space-below"></div>

        <div class="space-below"></div>

        <div class="article-style"></div>

    </div>


</div>

<footer class="site-footer">
    <div class="container">
        <p class="powered-by">

            Â© Blachly Lab, 2022 &middot; 

            Partially powered by the <a href="https://github.com/gcushen/hugo-academic" target="_blank">Academic theme</a> for <a href="http://gohugo.io" target="_blank">Hugo</a>.

            <span class="pull-right"><a href="#home" id="back_to_top"><span class="button_icon"><i class="fa fa-chevron-up fa-2x" aria-hidden="true"></i></span></a></span>

        </p>
    </div>
</footer>

<script src="//cdnjs.cloudflare.com/ajax/libs/gsap/1.18.4/TweenMax.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/gsap/latest/plugins/ScrollToPlugin.min.js"></script>




<script type="text/x-mathjax-config">
 MathJax.Hub.Config({ tex2jax: { inlineMath: [['$','$'], ['\\(','\\)']] } });
</script>

<script type="text/javascript" async
        src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS_CHTML">
</script>

</body>
</html>

